README.md

```diff
@@ -22,7 +22,7 @@ The app is built with [`create-react-app`](https://github.com/facebook/create-re
 
 ## Install
 
-With `node` >= 10 and `npm` >= 6.4.1 installed, run
+[![](https://byob.yarr.is/SgtPooki/ipfs-webui/node-10.x)](https://github.com/SgtPooki/ipfs-webui/actions/workflows/node-versions.yml) [![](https://byob.yarr.is/SgtPooki/ipfs-webui/node-12.x)](https://github.com/SgtPooki/ipfs-webui/actions/workflows/node-versions.yml) [![](https://byob.yarr.is/SgtPooki/ipfs-webui/node-14.x)](https://github.com/SgtPooki/ipfs-webui/actions/workflows/node-versions.yml) [![](https://byob.yarr.is/SgtPooki/ipfs-webui/node-16.x)](https://github.com/SgtPooki/ipfs-webui/actions/workflows/node-versions.yml) [![](https://byob.yarr.is/SgtPooki/ipfs-webui/node-18.x)](https://github.com/SgtPooki/ipfs-webui/actions/workflows/node-versions.yml)
```

Comment:
I tried a few variations of displaying the results, and this seems to be the cleanest. I'm open to ideas.

Reply:
Looks good, maybe add  context behind the badges:

```suggestion
With a supported Node.js version: [![](https://byob.yarr.is/SgtPooki/ipfs-webui/node-10.x)](https://github.com/SgtPooki/ipfs-webui/actions/workflows/node-versions.yml) [![](https://byob.yarr.is/SgtPooki/ipfs-webui/node-12.x)](https://github.com/SgtPooki/ipfs-webui/actions/workflows/node-versions.yml) [![](https://byob.yarr.is/SgtPooki/ipfs-webui/node-14.x)](https://github.com/SgtPooki/ipfs-webui/actions/workflows/node-versions.yml) [![](https://byob.yarr.is/SgtPooki/ipfs-webui/node-16.x)](https://github.com/SgtPooki/ipfs-webui/actions/workflows/node-versions.yml) [![](https://byob.yarr.is/SgtPooki/ipfs-webui/node-18.x)](https://github.com/SgtPooki/ipfs-webui/actions/workflows/node-versions.yml)
```